no3/style.css

```
@@ -0,0 +1,151 @@
+@import "reset.css";
+
+body {
+  background-color: #ecffdc;
+  margin: 0;
+  padding-top: 100px;
+}
+
+header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #4f7942;
+  margin: 0;
+  padding: 0;
+  padding-left: 40px;
+  padding-right: 40px;
```

padding을 좌우에만 주고싶은거면 요렇게 줘두 된다!!!ㅎㅎ

```suggestion
  padding: 0 40px;
```

---

맞아 나도 찾아보니까 padding: 0(상하) 40px(좌우); 이렇게 나타내는게 더 깔끔하고 가독성도 좋더라구..! margin에도 적용 가능!

---

왕... 넘 좋은방법..! 고마워잉🫶🏼